app/user_data.py

```diff
@@ -0,0 +1,14 @@
+import json
+import os
+
+
+path = os.path.abspath(os.path.join(os.path.dirname(__file__), "config.json"))
+
+
+def get_data() -> dict:
+    with open(path, "r") as text:
+        data = json.load(text)
+    return data
+
+
+FUEL_PRICE = float(get_data()["FUEL_PRICE"])
```

Comment:
Is `FUEL_PRICE` related to  user data?

Reply:
no, therefore it is used in the method as an imported constant. is it prolblem?

Reply:
It's important to keep  unrelated data in a separate file in order to enhance modularity and maintainability